package.json

```diff
@@ -39,6 +39,7 @@
     "@opentelemetry/api": "1.9.0",
     "@opentelemetry/semantic-conventions": "1.30.0",
     "debug": "4.4.0",
+    "ioredis": "^5.6.1",
```

Comment:
Let's not install this, it's a peer dependency only if you actually use redis. (check peerDependencies)